manager/src/test/java/com/redhat/service/smartevents/manager/api/user/ProcessorAPITest.java

```diff
@@ -467,7 +467,7 @@ public void addProcessorWithEmptyChannelParameterToBridge() {
         action.setMapParameters(params);
 
         Response response = TestUtils.addProcessorToBridge(bridgeResponse.getId(), new ProcessorRequest("myProcessor", null, null, action));
-        assertThat(response.getStatusCode()).isEqualTo(400);
+        assertThat(response.getStatusCode()).isEqualTo(202);
```

Comment:
This is actually a valid request according to the json schema provided by managed connectors